cmd/oras/internal/option/format.go

```diff
@@ -98,7 +99,11 @@ func (opts *Format) ApplyFlags(fs *pflag.FlagSet) {
 }
 
 // Parse parses the input format flag.
-func (opts *Format) Parse(_ *cobra.Command) error {
+func (opts *Format) Parse(cmd *cobra.Command) error {
+	// print deprecation message for table format
+	if opts.FormatFlag == "table" {
+		fmt.Fprint(cmd.ErrOrStderr(), "Format \"table\" is deprecated and will be removed in a future release.\n")
```

Comment:
Should we have a `Warning` prefix?

Reply:
Or, should we just use `logger.Warn()`?

Reply:
`Verbose` deprecation message does not have a warning prefix. I think it's better to keep the consistency.